tests/modules/token_embedders/elmo_token_embedder_test.py

```diff
@@ -0,0 +1,51 @@
+# pylint: disable=bad-continuation,no-self-use
+
+import os
+import numpy
+import h5py
+
+import torch
+from torch.autograd import Variable
+
+
+from allennlp.common.testing import AllenNlpTestCase
+from allennlp.modules.token_embedders.elmo_token_embedder import ELMoTokenEmbedder
+from allennlp.data.token_indexers.elmo_indexer import ELMoTokenCharactersIndexer
+from allennlp.data import Token, Vocabulary
+
+FIXTURES = os.path.join('tests', 'fixtures', 'elmo')
```

Comment:
I guess this is ok, but you could also override the `setUp()` of the test case and add it as an attribute there instead.

Reply:
This is a lighter weight way to allow other tests to use the same variable without having to define a new subclass.